symfony/framework-bundle/4.2/src/Kernel.php

```diff
@@ -37,7 +37,7 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
         $confDir = $this->getProjectDir().'/config';
 
         $loader->load($confDir.'/{packages}/*'.self::CONFIG_EXTS, 'glob');
-        $loader->load($confDir.'/{packages}/'.$this->environment.'/**/*'.self::CONFIG_EXTS, 'glob');
+        $loader->load($confDir.'/{packages}/'.$this->environment.'/*'.self::CONFIG_EXTS, 'glob');
```

Comment:
shouldn't we do the same for 3.3 recipe?

Reply:
updated for 3.3 also